src/gbastd/types/time.h

```diff
@@ -0,0 +1,41 @@
+#ifndef TIME_H
+#define TIME_H
+
+#include "u.h"
+#include "../../defines.h"
+
+typedef struct
+{
+    u8 seconds;
+    u8 minutes;
+    u8 hours;
+} time;
+
+u64 tick = 0;
+
+void addTick()
+{
+    tick += UPDATES_PER_SECOND;
```

Comment:
As I said, UPDATES_PER_SECOND are used only to calculate `time`. As we already have discuss tick contains ticks, and addTick simply should accept pointer to timer struct, and increase tick variable in it(And check if tick is passed over u64 limit, but I am not sure if it matters, cause unsigned int is like from 0..N, looks like integer limit there do not really matter, but still try to check if).